### ISSUE TITLE
Multipatch view support

### ISSUE THREAD

**Issue by mgsx-dev** [b2renger/PdDroidPublisher#13]
Prototype is concluant and usable. But implementation is not perfect yet : 
- all patches must be in the same directory (or subdirectory) as the main patch
- overrides and theme may be configurable by patch view
- array timer update is done by view and not only the current view (need optimizations)
- API is not well documented for this use.
- test example is very simple and maybe there still some issues.

**Comment by b2renger** [comment]
If we want to use seperate views and with duplicate guis in at least two of this views, objects doesn't get updated in the view that is not visible.

Test case with aciddrums example, using 3 view (as discussed): full stack, sequencer and controls. Changing any parameters in sequencer and controls will not be repercuted on the full stack view.

This case will also be problematic with the persist state-saving system as you'll have several gui objects with the same send and receive names.

**Comment by mgsx-dev** [comment]
persistence shouldn't be a problem : 
- when saving, last received value in Pd (from any widget) is saved (that's what we want)
- when loading, all widget having the same receiver name will be updated (that's what we want too).

The only problem is array, because Pd doesn't allow multiple arrays with the same name. That's a limitation.

In fact widget clones problem is not really linked to multipatch view, you can reproduce this with a single patch and 2 slider with same send and receive name. And you can reproduce it in Pd Desktop as well.

I think widget sync wouldn't be nice because we can't implement it in Pd Desktop. To do it, i think you have to implement it in Pd like example provided in attachment even if it's cumbersome to do so ...
[SliderSyncExample.pd.txt](https://github.com/b2renger/PdDroidPublisher/files/12516/SliderSyncExample.pd.txt)

In conclusion, I think GUI duplication should be avoided for now. If it's really needed, Pd implementation is the way to go.

What do you think ?

**Comment by b2renger** [comment]
yes I agree with you conclusion, if done it needs to be done in pd desktop aswell.  I'll add it to the todoc list :)

**Comment by mgsx-dev** [comment]
feature working properly with some limitations todoc.

**Comment by b2renger** [comment]
documented in the next release of the website in the post How_to_configure_your_launcher
